.github/workflows/helm-docs.yaml

```diff
@@ -0,0 +1,45 @@
+name: Create helm-docs
+
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
+        with:
+          app_id: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_ID }}
+          private_key: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_PEM }}
```

Comment:
I right now used the GitHub App, but will propose some naming conventions for apps per repo or role 🤔